src/pwr.rs

```diff
@@ -236,4 +238,69 @@ impl Pwr {
 
         VoltageScale::Scale1
     }
+
+    #[cfg(feature = "revision_v")]
+    pub fn freeze_vos0(self, rcc: &Rcc, syscfg: &SYSCFG) -> VoltageScale {
+        // NB. The lower bytes of CR3 can only be written once after
+        // POR, and must be written with a valid combination. Refer to
+        // RM0433 Rev 7 6.8.4. This is partially enforced by dropping
+        // `self` at the end of this method, but of course we cannot
+        // know what happened between the previous POR and here.
+
+        #[cfg(any(feature = "singlecore"))]
+        self.rb.cr3.modify(|_, w| {
+            w.scuen().set_bit().ldoen().set_bit().bypass().clear_bit()
+        });
+
+        #[cfg(any(feature = "dualcore"))]
+        self.rb.cr3.modify(|_, w| {
+            use SupplyConfiguration::*;
+
+            match self.supply_configuration {
+                LDOSupply => w.sden().clear_bit().ldoen().set_bit(),
+                DirectSMPS => w.sden().set_bit().ldoen().clear_bit(),
+                SMPSFeedsIntoLDO1V8 => unsafe {
+                    w.sden().set_bit().ldoen().set_bit().sdlevel().bits(1)
+                },
+                SMPSFeedsIntoLDO2V5 => unsafe {
+                    w.sden().set_bit().ldoen().set_bit().sdlevel().bits(2)
+                },
+                Bypass => {
+                    w.sden().clear_bit().ldoen().clear_bit().bypass().set_bit()
+                }
+                Default => {
+                    // Default configuration. The actual reset value of
+                    // CR3 varies between packages (See RM0399 Section
+                    // 7.8.4 Footnote 2). Therefore we do not modify
+                    // anything here.
+                    w
+                }
+            }
+        });
+        // Verify supply configuration, panics if these values read
+        // from CR3 do not match those written.
+        #[cfg(any(feature = "dualcore"))]
+        self.verify_supply_configuration();
+
+        // Validate the supply configuration. If you are stuck here, it is
+        // because the voltages on your board do not match those specified
+        // in the D3CR.VOS and CR3.SDLEVEL fields.  By default after reset
+        // VOS = Scale 3, so check that the voltage on the VCAP pins =
+        // 1.0V.
+        while self.rb.csr1.read().actvosrdy().bit_is_clear() {}
+
+        // We have now entered Run mode. See RM0433 Rev 7 Section 6.6.1
+
+        // go to VOS1 voltage scale for high performance
+        self.rb.d3cr.write(|w| unsafe { w.vos().bits(0b11) });
+        while self.rb.d3cr.read().vosrdy().bit_is_clear() {}
+
+        // Enable overdrive for maximum clock
+        // Syscfgen required to set enable overdrive
+        rcc.rb.apb4enr.modify(|_, w| w.syscfgen().enabled());
+        unsafe { syscfg.pwrcr.modify(|_, w| w.oden().bits(1)) };
```

Comment:
You probably want `w.oden().set_bit()` here

Reply:
Doesn't exist for this register for some reason.
```
unsafe { syscfg.pwrcr.modify(|_, w| w.oden().set_bit()) };
    |                                                      ^^^^^^^ method not found in `stm32h7::stm32h743v::syscfg::pwrcr::ODEN_W<'_>`
```

Reply:
Ah the problem is it does exist on some parts 
https://docs.rs/stm32h7/0.11.0/stm32h7/stm32h747cm7/syscfg/pwrcr/struct.ODEN_W.html#method.set_bit
But not on others
https://docs.rs/stm32h7/0.11.0/stm32h7/stm32h743/syscfg/pwrcr/struct.ODEN_W.html#method.set_bit

The `bits` method is the opposite, which is why the CI is failing on it.

Ultimately it would be worth extending this PR https://github.com/stm32-rs/stm32-rs/pull/366/files

But for now you can write
```rust
#[cfg(any(feature = "dualcore"))]
syscfg.pwrcr.modify(|_, w| w.oden().set_bit());
#[cfg(not(any(feature = "dualcore")))]
unsafe { syscfg.pwrcr.modify(|_, w| w.oden().bits(1)) };
```